funding/manager.go

```diff
@@ -601,6 +614,16 @@ func (m *Manager) BatchChannelSetup(batch *order.Batch,
 		}
 	}
 
+	// If the only order that we were involved in is a sidecar channel and
+	// we are the provider, not the ultimate recipient, then there's nothing
+	// to wait for since the channel is going to be opened to another node.
+	if len(chanPoints) == 0 && someOrdersAreSidecars {
```

Comment:
Isn't it enough to check `len(chanPoints) == 0`?

Reply:
Yeah I think so, I guess this makes it slightly more explicit? Can't think of any other case where we _wouldn't_ have any chan points at this point though.